packages/styling/src/styles/theme.ts

```diff
@@ -108,21 +108,30 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
-  const typography = { ...DefaultTypography, ...theme.typography };
+  let typography = { ...DefaultTypography };
+  if (theme.typography) {
+    typography = {
+      families: { ...DefaultTypography.families, ...theme.typography.families },
+      weights: { ...DefaultTypography.weights, ...theme.typography.weights },
+      sizes: { ...DefaultTypography.sizes, ...theme.typography.sizes },
+      types: { ...DefaultTypography.types, ...theme.typography.types }
+    } as ITypography;
```

Comment:
That's odd, does it complain about the type here? It looks like it shouldn't.

Reply:
Casting should be by exception and for good reason as this block will lose type safety.